src/test/java/com/google/api/codegen/protoannotations/testdata/java_library.baseline

```diff
@@ -4500,19 +4497,6 @@ public class LibraryClient implements BackgroundResource {
   private static final PathTemplate SHELF_PATH_TEMPLATE =
       PathTemplate.createWithoutUrlEncoding("shelves/{shelf_id}");
 
-  /**
-   * Formats a string containing the fully-qualified path to represent
-   * a archive_book resource.
-   *
-   * @deprecated Use the {@link ArchiveBookName} class instead.
-   */
-  @Deprecated
-  public static final String formatArchiveBookName(String archive, String book) {
```

Comment:
The `archive_book` resource name was created through the "Trie" strategy. We stop generate any resource name helpers for this pattern.